python/benchmarks/finalize_staged_data.py

```diff
@@ -74,32 +74,36 @@ def peakmem_finalize_staged_data(self, cache:CachedDFGenerator, param:int):
     def teardown(self, cache:CachedDFGenerator, param:int):
         self.ac.delete_library(self.lib_name)
 
-class FinalizeStagedDataWiderDataframeX3(FinalizeStagedData):
-    '''
-        The test is meant to be executed with 3 times wider dataframe than the base test
-    '''
+if sys.version_info >= (3, 8):
+    ## asv 0.6.0 or later is required for skipping (Python > 3.6)
```

Comment:
This looks like a temporary fix until #2057 is merged. Can't we wait for that to get merged first and not worry about python 3.6?

Reply:
This is in order to see results for benchmarks. They will fail because of guarded code uses something not avail in python 3.6. We should have it until workflow is upgraded to use it instead of 3.6